Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -101,8 +101,8 @@ def bottle
     end
 
     ensure_relocation_formulae_installed! unless args.skip_relocation?
-    args.named.to_resolved_formulae(uniq: false).each do |f|
-      bottle_formula f, args: args
+    args.named.zip(args.named.to_resolved_formulae(uniq: false)).each do |fn, f|
+      bottle_formula f, fn, args: args
```

Comment:
```suggestion
    args.named.zip(args.named.to_resolved_formulae(uniq: false)).each do |filename, f|
      bottle_formula f, filename, args: args
```

assuming that's what `fn` means?

Reply:
You're right, that's unclear. I've updated it with a more clear variable name. This new version is also more careful about assigning it, so it should still work in the case the name of a formula gets passed in.